packages/react-native-editor/android/app/src/main/java/com/gutenberg/MainApplication.java

```diff
@@ -240,6 +240,11 @@ public void gutenbergDidSendButtonPressedAction(String buttonType) {
 
             }
 
+            @Override
+            public void requestPreview() {
+
```

Comment:
Want to do something like https://github.com/WordPress/gutenberg/blob/27fc8980ddf875757e671688f83a9bfbfa73a0c5/packages/react-native-editor/android/app/src/main/java/com/gutenberg/MainApplication.java#L235 and put out a Toast popup message here?

Reply:
Yeah, I like the idea of notifying somehow that a mocked native call has been called, it could be useful when testing on the demo project 👍 .

Reply:
I applied the suggestion in [this commit](https://github.com/WordPress/gutenberg/pull/33266/commits/2bee10d420c893aa4f1d883be1143bfeccea222e).